### PR TITLE
Update mac-resources.md

### PR DESCRIPTION
**Pull request by msbemba**
Update output types to json and table

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6700

File: microsoft-365/security/defender-endpoint/mac-resources.md [hunk 1]
```diff
@@ -82,6 +82,13 @@ There are several ways to uninstall Microsoft Defender for Endpoint on macOS. No
 
 - Open **Finder > Applications**. Right click on **Microsoft Defender for Endpoint > Move to Trash**.
 
+### Supported output types
+
+Supports table and JSON format output types. For each command, there's a default output behavior. You can modify the output in your preferred output format using the following commands:
+
+-output json
+-output table
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-resources.md]:
```suggestion
`-output json`

`-output table`
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested change if you agree with it.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**